apps/fz_http/lib/fz_http/api_tokens.ex

```diff
@@ -0,0 +1,53 @@
+defmodule FzHttp.ApiTokens do
+  @moduledoc """
+  The ApiTokens context.
+  """
+
+  import Ecto.Query, warn: false
+  alias FzHttp.Repo
+
+  alias FzHttp.ApiTokens.ApiToken
+
+  def count, do: count(ApiToken)
+  def count(nil), do: nil
+
+  def count(queryable) when is_struct(queryable) or is_atom(queryable),
+    do: Repo.aggregate(queryable, :count)
+
+  def count(user_id), do: count(from(a in ApiToken, where: a.user_id == ^user_id))
```

Comment:
I think pattern matching makes this code not explicit and less extendable - what if we then want to count by account? We won't be able to match on type anymore.
```suggestion
  def count, do: Repo.aggregate(ApiToken, :count)

  def count_by_user_id(user_id), do: count(from(a in ApiToken, where: a.user_id == ^user_id))
```

Reply:
Agree. Sometimes I get a little carried away ✈️